functions/src/api/v1/rooms/joinRoom.ts

```diff
@@ -65,13 +72,27 @@ export const joinRoom = wrap(async (req: Request, res: Response) => {
     if (room.password !== roomPassword && !isCurrentAdmin) {
         throw new ForbiddenError('Wrong password to join the room')
     }
+    if (room?.status === StatusEnded) {
```

Comment:
It could be an enhancement to use functions to give room status. For example something like this

```
isStatusEnded(room: Room) => room.status === statusEnded
```

Reply:
idealy, having room.isStatusEnded will be great too, thus having a real Room class